### DISCUSSION TITLE
Bidirectional Path Tracer

### DISCUSSION THREAD

**Discussion by Tscho1995** [mitsuba-renderer/mitsuba3#427] [General]
Are there any plans to implement the Bidirectional Path Tracer in Mitsuba3? It used to be in Mitsuba 0.5.0.

All the best,

jonathan

**Comment by wjakob** [comment]
We don't plan to work on one, but outside contributions are welcomed.

**Reply by Tscho1995** [reply]
All right, thanks for your reply. We plan to implement a line lighting emitter, to render Laser Triangulation processes. Since the emitter is really small, i think the only way to render in an appropriate time is to use the bidirectional path Tracer. Or can you recommend a different approach?
